partner integration docs v0 (#6262)
ClickHouse/clickhouse-docs: docs/integrations/index.mdx

| Partner integrations | Built or maintained, and supported by, third-party software vendors |
| Community integrations | Built or maintained and supported by community members. No direct support is available besides the public GitHub repositories and community Slack channels |

> :::tip Building a partner integration?

Reviewer: @Amehla we'll need the stubbed link for the partner portal and sign up page

> If you are building a product that connects to ClickHouse, start with the [Integration development](/integrations/integration-development) guides: [building integrations](/integrations/integration-development/building-integrations), [testing](/integrations/integration-development/testing-your-integration), and [documenting your product](/integrations/integration-development/documenting-your-integration). When the [partner portal](https://clickhouse.com/partners) is live, [sign up](https://clickhouse.com/partners) to register your integration.

Reviewer: ```suggestion
If you are building a product that connects to ClickHouse, start with the [Integration development](/integrations/integration-development) guides: [building integrations](/integrations/integration-development/building-integrations), [testing](/integrations/integration-development/testing-your-integration), and [documenting your product](/integrations/integration-development/documenting-your-integration). Sign up in our [partner portal](https://clickhouse.com/partners) to register your integration.
```

> If you are building a product that connects to ClickHouse, start with the [Integration development](/integrations/integration-development) guides: [building integrations](/integrations/integration-development/building-integrations), [testing](/integrations/integration-development/testing-your-integration), and [documenting your product](/integrations/integration-development/documenting-your-integration). When the [partner portal](https://clickhouse.com/partners) is live, [sign up](https://clickhouse.com/partners) to register your integration.

Reviewer 2: ```suggestion
:::tip[Building a ClickHouse integration?]
If you are building a product that connects to ClickHouse, start with the [Integration development](/integrations/integration-development) guides: [building integrations](/integrations/integration-development/building-integrations), [testing](/integrations/integration-development/testing-your-integration), and [documenting your product](/integrations/integration-development/documenting-your-integration). When the [partner portal](https://clickhouse.com/partners) is live, [sign up](https://clickhouse.com/partners) to register your integration.
```
